src/view/screens/Search/Search.tsx

```diff
@@ -786,23 +769,28 @@ export function SearchScreen(
           ) : undefined}
         </View>
 
-        {query || inputIsFocused ? (
+        {(query || inputIsFocused) && (
           <View style={styles.headerCancelBtn}>
             <Pressable
               onPress={onPressCancelSearch}
               accessibilityRole="button"
               hitSlop={HITSLOP_10}>
               <Text style={[pal.text]}>
-                <Trans>Cancel</Trans>
+                {isWeb && !inputIsFocused && q ? (
+                  <Trans>Back</Trans>
+                ) : (
+                  <Trans>Cancel</Trans>
+                )}
```

Comment:
If there's history for us to go back into, we could show "Back" here instead or "Cancel". Not sure if we want to though.